packages/augur-core/source/contracts/ZeroXExchange.sol

```diff
@@ -95,6 +96,8 @@ contract ZeroXExchange is IExchange, ReentrancyGuard {
     // Orders with specified senderAddress and with a salt less than their epoch are considered cancelled
     mapping (address => mapping (address => uint256)) public orderEpoch;
 
+    Registry public registry;
```

Comment:
The idea for the 0x exchange is the predicate will use its own custom version on main chain?